postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -153,6 +153,21 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
           r <- testResult.runCollect
         } yield assert(r.head)(equalTo(expected))
 
+        assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
+      },
+      testM("to_ascii") {
+        val query = select(ToAscii("Karél")) from customers
+
+        val testResult = execute(query).to[String, String](identity)
+
+        val assertion = testResult.runCollect.fold(
```

Comment:
you should be able to use `fails` instead of `fold` to get rid of `assert(true)(isFalse)`
something like:
```scala
val assertion = assert(testResult.runCollect.mapError(_.getMessage))(fails(equalTo("ERROR: encoding conversion from UTF8 to ASCII not supported")))
```
there are good examples of this in the main zio repository tests